lib/main_common.pm

```diff
@@ -2576,7 +2576,7 @@ sub load_hypervisor_tests {
         my $modules = $feature->{modules};
         my $hypervisor = $feature->{hypervisor};
         if ($test eq 'ENABLE_SRIOV_NETWORK_CARD_PCI_PASSTHROUGH') {
-            next unless is_sle('>15');
+            next unless is_sle('>15') && !is_sle('=15-sp2');
```

Comment:
Does sriov tests run on sle15sp1 for MU tests? if not, we can run on `>=15-sp3`

Reply:
15sp1 has ended, so no triggering of test at all. Both will have the same effect ;-)

Reply:
Service Pack Release	FCS Date	General Ends	LTSS Ends	LTSS Core Ends
SUSE Linux Enterprise Server 15	16 Jul 2018	31 Dec 2019	31 Dec 2022	 
SUSE Linux Enterprise Server 15 SP1	24 Jun 2019	31 Jan 2021	31 Jan 2024	 
SUSE Linux Enterprise Server 15 SP2	21 Jul 2020	31 Dec 2021	31 Dec 2024
from: https://www.suse.com/lifecycle/#suse-linux-enterprise-server-15

@alice-suse What's your opinion on this?

Reply:
15sp2 has TERADATA special customer case, so it is actually alive, and the sle15.[0-1] have really ended. We should test 15sp2, although currently we have to skip it as temporary workaround. So, I personally prefer current way, but better add a comment to explain why 15sp2 is skipped -- add the bug number.